src/codemagic/firebase/api_client.py

```diff
@@ -0,0 +1,20 @@
+from googleapiclient import discovery
+from oauth2client.service_account import ServiceAccountCredentials
+
+from .resource_managers.release_manager import FirebaseReleaseResourceManager
+
+
+class FirebaseAPIClient:
```

Comment:
Elsewhere `API` has not been written in caps within class names. Instead it is like `SomeApiClient`.